main/webapp/modules/core/langs/translation-es.json

```diff
@@ -626,43 +626,43 @@
 	},
 	"core-buttons": {
 		"cancel": "Cancelar",
-		"ok": "&nbsp;&nbsp;OK&nbsp;&nbsp;",
+		"ok": "&nbsp;&nbsp;Aceptar&nbsp;&nbsp;",
 		"import-proj": "Importar projecto",
 		"select-all": "Seleccionar todos",
 		"unselect-all": "Seleccionar ninguno",
-		"deselect-all": "De-select All",
+		"deselect-all": "De-seleccionar todos",
 		"select": "Seleccionar",
 		"unselect": "Quitar",
 		"startover": "&laquo; Inicio",
 		"conf-pars-opt": "Configurar opciones de carga &raquo;",
-		"reselect-files": "&laquo; Re-select Files",
+		"reselect-files": "&laquo; Re-seleccionar Archivos",
 		"create-project": "Crear proyecto &raquo;",
 		"next": "Siguiente &raquo;",
 		"add-url": "Agregar otra URL",
 		"update-preview": "Actualizar&nbsp;vista&nbsp;previa",
-		"pick-record": "Pick Record Elements",
+		"pick-record": "Escojer elementos del registro",
 		"merge-cluster": "Unir seleccionados y reagrupar",
 		"merge-close": "Unir seleccionados y cerrar",
 		"close": "Cerrar",
-		"reset-template": "Reset Template",
+		"reset-template": "Restaurar Plantilla",
 		"export": "Exportar",
 		"preview": "Vista previa",
 		"download": "Descargar",
 		"upload": "Cargar",
 		"apply": "Aplicar",
-		"enter": "Enter",
-		"esc": "Esc",
+		"enter": "Aceptar",
+		"esc": "Cancelar",
 		"refresh": "Actualizar",
 		"reset-all": "Restaurar todos",
 		"remove-all": "Remover todos",
-		"perform-op": "Perform Operations",
+		"perform-op": "Ejecutar Operaciones",
```

Comment:
What's the context of this one?  I mean, "Ejecutar operaciones" sounds better to me, but I'm not sure about the context it is displayed in.